File: frontend/classes/Turn.ts
interface TurnData {
  id: number
  index: number
  game: number
}

class Turn {
  id: number
  index: number
  game: number

  constructor(data: TurnData) {
    this.id = data.id
    this.index = data.index
    this.game = data.game
  }
}

export default Turn

File: mvp-backend/rorapp/models/turn.py
from django.db import models
from rorapp.models.game import Game


# Model for representing turns
class Turn(models.Model):
    index = (
        models.PositiveIntegerField()
    )  # 1-based index because it's a user-facing value
    game = models.ForeignKey(Game, on_delete=models.CASCADE)
